pkgs/applications/audio/openutau/default.nix

```diff
@@ -37,11 +37,11 @@ buildDotnetModule rec {
   # socket cannot bind to localhost on darwin for tests
   doCheck = !stdenv.isDarwin;
 
-  # net7.0 replacement needed until upstream bumps to dotnet 7
+  # net8.0 replacement needed until upstream bumps to dotnet 8
   postPatch = ''
     substituteInPlace OpenUtau/OpenUtau.csproj OpenUtau.Test/OpenUtau.Test.csproj --replace \
       '<TargetFramework>net6.0</TargetFramework>' \
-      '<TargetFramework>net7.0</TargetFramework>'
+      '<TargetFramework>net8.0</TargetFramework>'
```

Comment:
Is there a reason why we're patching this? .NET 6 is an LTS release and isn't EOL yet.

Reply:
.NET 6 just went EOL as of Nov. 12. As of now, the defaults for `dotnet-sdk` and `dotnet-runtime` still point to .NET 6, but this will likely change soon. I have no idea how long it will take for upstream to update to .NET 8. I would say that the patch is appropriate in this case, but I'm open to further discussion.

Reply:
The upstream maintainer made a PR that updates to .NET 8: https://github.com/stakira/OpenUtau/pull/1331.